<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(215, 215, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_edit_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
qproperty-alignment: 'AlignVCenter | AlignRight'
border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graphic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Osaka</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>Graphic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_result">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 195, 255);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>270</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>110</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>350</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>430</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_to">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>110</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>190</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>110</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_e">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>110</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 180, 255);
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_atan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_acos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_so">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ln">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sz">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_po">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mod">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sqrt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>439</y>
      <width>100</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgb(210, 195, 255);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mu">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_di">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_x">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox { 
	font: 14pt &quot;Osaka&quot;;
	background-color: white;
}
QLabel { 
	font: 14pt &quot;Osaka&quot;;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox { 
	font: 14pt &quot;Osaka&quot;;
	background-color: white;
}
QLabel { 
	font: 14pt &quot;Osaka&quot;;
}</string>
    </property>
    <property name="text">
     <string>x =</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_end">
    <property name="geometry">
     <rect>
      <x>841</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.989999994635582</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>Min x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>Max x</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_begin">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 16pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.989999994635582</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_c">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 195, 255);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_credit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Osaka</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgba(122, 129, 255, 195);
}</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ac">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>439</y>
      <width>100</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	font: 18pt &quot;Osaka&quot;;
	background-color : rgb(210, 195, 255);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
